src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+
+    public Circle(String color, double radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public void print() {
+        System.out.println("Figure: circle, area: " + calculateArea() + " sq.units, color: "
+                + getColor() + ", radius: " + getRadius());
```

Comment:
```suggestion
                + getColor() + ", radius: " + radius);
```
you have direct access to fields of the class